### DISCUSSION TITLE
[Announcement] Additional Announcement for Threat Hunt

### DISCUSSION THREAD

**Discussion by sujin0529** [prosyslab-classroom/cs348-information-security#466] [Announcements]
Hi, all
We are thankfully receiving a lot of reports about threat hunt.
But, we would like to announce some restrictions to keep this activity going for a long time.

- Do not send the report with the same root cause.
  - We'll grade it with your first report if you send multiple reports of the same root cause.
- Do not upload data used in threat hunt activity outside the KAIST domain.
- Do not practice DDoS Attack.
- Do not falsify the DB data.

Thanks for participating in the threat hunt.

**Comment by KihongHeo** [comment]
Please merge this message into #213 and close this.
